nearcore/src/config.rs

```diff
@@ -1233,11 +1233,11 @@ struct NodeKeyFile {
 }
 
 impl NodeKeyFile {
-    fn from_file(path: &Path) -> Self {
-        let mut file = File::open(path).expect("Could not open key file.");
+    fn from_file(path: &Path) -> std::io::Result<Self> {
+        let mut file = File::open(path)?;
         let mut content = String::new();
-        file.read_to_string(&mut content).expect("Could not read from key file.");
-        serde_json::from_str(&content).expect("Failed to deserialize KeyFile")
+        file.read_to_string(&mut content)?;
+        Ok(serde_json::from_str(&content)?)
     }
```

Comment:
I would add `with_context(||format!("path={:?}", path)`, or something similar to indicate, which file couldn't be opened.

Reply:
Yeah, I made the caller in load_config() attach that context with the pathname. Felt more natural to me to do it that way since the caller knows what path it tried to open. No super strong opinion about that though. Do you think its better to add the context here?

Reply:
Either way is fine. It depends on how many callers are there, if this function is only called 1-3 times, then doing it on the caller side is fine.